package.json

```diff
@@ -17,6 +17,7 @@
     "can-stache-bindings": "^3.0.5",
     "can-types": "^1.0.0",
     "can-util": "^3.7.0",
+    "can-validate-interface": "^0.1.0",
```

Comment:
can-validate-interface only has a 0.1.0 release at the moment. I could release a 1.0.0, but I think that setting this version to `0.x` would work just as well. Thoughts?